src/Robo/Commands/Acsf/AcsfCommand.php

```diff
@@ -34,22 +34,20 @@ public function printPreamble() {
    * @aliases acsf acsf:init
    * @options acsf-version
    */
-  public function acsfInitialize($options = ['acsf-version' => '^1.33.0']) {
+  public function acsfInitialize() {
     $this->printPreamble();
     $this->acsfHooksInitialize();
     $this->say('Adding acsf module as a dependency...');
-    $package_options = [
-      'package_name' => 'drupal/acsf',
-      'package_version' => $options['acsf-version'],
-    ];
-    $this->invokeCommand('internal:composer:require', $package_options);
-    $this->say("In the future, you may pass in a custom value for acsf-version to override the default version. E.g., blt recipes:acsf:init:all --acsf-version='8.1.x-dev'");
     $this->acsfDrushInitialize();
     $this->say('Adding acsf-tools drush module as a dependency...');
-    $package_options = [
-      'package_name' => 'acquia/acsf-tools',
-      'package_version' => '^8.1',
-    ];
+    $result = $this->taskExecStack()
+      ->dir($this->getConfigValue('repo.root'))
+      ->exec("composer bin drush-8 require acquia/acsf-tools")
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+    if (!$result->wasSuccessful()) {
+      throw new BltException("Unable to require acquia/acsf-tools");
+    }
     $this->invokeCommand('internal:composer:require', $package_options);
```

Comment:
`$package_options` is undefined in the change are you meaning to remove the `invokeCommand`?